bgcompute.py

```diff
@@ -4,13 +4,15 @@
 from flask import Flask
 
 from arlo_server import app, db
-from arlo_server.models import RoundContest
+from arlo_server.models import RoundContest, Contest
 from arlo_server.routes import compute_sample_sizes
 
 
 def bgcompute():
-    # round contests that don't have sample_size_options
-    round_contests = RoundContest.query.filter_by(sample_size_options=None)
+    # Round contests for targeted contests that don't have sample_size_options
+    round_contests = RoundContest.query.filter_by(Contest.is_targeted).filter_by(
```

Comment:
sneaking in an unrelated change? ;)

Reply:
whoops, that was from my earlier poking around. thanks for catching, wasn't supposed to be in here